CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Unreleased
 
+# v2.0.0
+
+[Documentation](https://github.com/awslabs/mountpoint-s3-csi-driver/blob/v2.0.0/README.md)
+
+### Notable changes
```

Comment:
Would add a 'breaking changes' section here to make it obvious

Reply:
I'll update in a follow-up PR.